packages/odyssey-react/src/components/Tag/Tag.module.scss

```diff
@@ -10,22 +10,22 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-.list {
+.root {
   display: inline-block;
-  margin-block-end: -$spacing-xs;
-  margin-inline-start: $spacing-xs;
+  margin-block-end: calc(-1 * var(--RootSpaceMarginBlockEnd));
+  margin-inline-start: var(--RootSpaceMarginInlineStart);
   list-style-type: none;
 }
 
 .tag {
```

Comment:
consider renaming the class to `.item` here to better align with the theme API we're exposing